docs/libertyStatus.md

```diff
@@ -1,3 +1,6 @@
 ## libertyStatus task  
 The `libertyStatus` task checks and prints whether the server is running or not.  
 This task should automatically call `libertyCreate` and create the server if it does not yet exist.
+
+### dependsOn
+`libertyStatus` depends on `libertyCreate`.
```

Comment:
I guess you could remove that last sentence prior:  This taskshould automatically call `libertyCreate` and create the server if it does not exist.  ?